pkg/util/validation/limits.go

```diff
@@ -663,6 +666,11 @@ func (o *Overrides) CompactorBlockUploadVerifyChunks(tenantID string) bool {
 	return o.getOverridesForUser(tenantID).CompactorBlockUploadVerifyChunks
 }
 
+// CompactorBlockUploadMaxBlockSizeBytes returns the maximum size in bytes of a block that is allowed to be uploaded or validated.
+func (o *Overrides) CompactorBlockUploadMaxBlockSizeBytes(tenantID string) int64 {
```

Comment:
Odd fluctuation here between tenantID and userID everywhere else, I'll look into it.